test/e2e-bootstrap/bootstrap_test.go

```diff
@@ -695,3 +698,30 @@ func containsGVK(objs []runtime.Object, gvk schema.GroupVersionKind) bool {
 	}
 	return false
 }
+
+func waitForFeatureGates(t *testing.T, featureGateAccess featuregates.FeatureGateAccess) {
+	select {
+	case <-featureGateAccess.InitialFeatureGatesObserved():
+		fg, err := featureGateAccess.CurrentFeatureGates()
+		require.NoError(t, err, "unable to get initial features")
+		enabled, disabled := getEnabledDisabledFeatures(fg)
+		t.Logf("FeatureGates initialized: enabled=%v  disabled=%v", enabled, disabled)
+	case <-time.After(1 * time.Minute):
+		t.Fatalf("timed out waiting for FeatureGate detection")
+	}
+}
+
+func getEnabledDisabledFeatures(features featuregates.FeatureGate) ([]string, []string) {
```

Comment:
suggestion: There are some other places where we do this using a `set` type instead, such as here: https://github.com/openshift/machine-config-operator/blob/master/test/helpers/utils.go#L1163-L1201

The rationale behind doing it that way is that we can provide very useful and descriptive error messages if one or more required FeatureGates are not enabled. We may also want to hoist the `validateFeatureGatesEnabled()` function as well, but I will leave that to your discretion :smile: